### PR TITLE
Signing jobs

### PR DESCRIPTION
**Pull request by IsaacYangSLA**

### CODE REVIEW

File: nvflare/lighter/utils.py [hunk 1]
```diff
@@ -12,21 +12,109 @@
...
+                padding=padding.PSS(
+                    mgf=padding.MGF1(hashes.SHA256()),
+                    salt_length=padding.PSS.MAX_LENGTH,
+                ),
+                algorithm=hashes.SHA256(),
+            )
+            signatures[file] = b64encode(signature).decode("utf-8")
+        for folder in folders:
+            signature = signing_pri_key.sign(
+                data=folder.encode("utf-8"),
```
Comment (yhwen) [inline nvflare/lighter/utils.py:70]: Should we recursively sign the files content within a folder? or simply sign the folder.encode("utf-8") data?

Comment (IsaacYangSLA) [reply nvflare/lighter/utils.py:70]: The os.walk does the recursive traversal into each folder.  The content of the folder, ie files and subfolders, are signed and the signature file is inside the folder.  So files are signed by contents and subfolders are signed by folder names.

**Review by yhwen — APPROVED** [review]
LGTM

**Pull request merged**